awx/api/views/__init__.py

```diff
@@ -405,10 +405,8 @@ class InstanceInstanceGroupsList(InstanceGroupMembershipMixin, SubListCreateAtta
     relationship = 'rampart_groups'
 
     def is_valid_relation(self, parent, sub, created=False):
-        if parent.node_type == 'control':
-            return {'msg': _(f"Cannot change instance group membership of control-only node: {parent.hostname}.")}
-        if parent.node_type == 'hop':
-            return {'msg': _(f"Cannot change instance group membership of hop node: {parent.hostname}.")}
+        if parent.node_type == 'control' or parent.node_type == 'hop' or (sub.name == 'controlplane' and parent.node_type == 'hybrid'):
+            return {'msg': _(f"Cannot change instance group membership of {parent.node_type} node: {parent.hostname}.")}
```

Comment:
This message will say the wrong thing for hybrid nodes. You _can_ change memberships of hybrid nodes, but not for the control plane. @jbradberry along the lines of your recent comments.

The sneaky thing here is that instance-instance-group endpoints use `is_valid_relation` symmetrically, on attach and unattach, and no other types of views do this.

It may still be possible to resolve the issue with this approach, but need the wording to be right, and use the setting instead of hard-coding "controlplane"